danndav/bhankas
# #165363233:work on lfa feedback

# What does this PR do?
This PR works on  LFA feedbacks

# Description of Task to be completed?
remove banker test from the landing page footer
remove Home navigation on landing page
add pagination to the account page
add modal on each page
work on Header on each page
add the name of staff on staff page..eg welcome David Imodoye
check for better avatar for profile picture and remove background
add cashier page
add search button on cashier page
make the Banka title logo Bolder
remove the plus(+) sign from the add admin nav link

# How should this be manually tested?
git fetch and checkout to bg-165363233-work-feedbacks-base-lfa-review 
checkout to the branch
run the application
navigate to /tasks page
click on any work schedule to check changes

# Any background context you want to provide?
N/A

# What are the relevant pivotal tracker stories?
#165363233
# screenshots (if appropriate)
N/A
# Questions:
N/A\
